database/src/main/java/database/service/SqlDatabaseService.java

```diff
@@ -440,4 +479,46 @@ private <T extends BaseEntity> T createAndFillEntity(Class<? extends BaseEntity>
         }
         return entity;
     }
+
+    private void closeConnectionStatementResources(Connection connection, Statement statement) {
+        if (statement != null) {
+            try {
+                statement.close();
+            } catch (SQLException e) {
+                LOG.warn("Unable to close Statement: {}", e.getMessage());
+            }
+        }
+        if (connection != null) {
+            try {
+                connection.close();
+            } catch (SQLException e) {
+                LOG.warn("Unable to close Connection: {}", e.getMessage());
+            }
+        }
+    }
+
+    private void closeConnectionStatementResultSetResources(Connection connection, Statement statement,
+                                                            ResultSet resultSet) {
+        if (statement != null) {
```

Comment:
Please reuse `closeConnectionStatementResources(Connection connection, Statement statement)` to close connection and statement and to do not duplicate code.